frontend/vre/index.html

```diff
@@ -12,6 +12,7 @@
         >
         <link rel="stylesheet" type="text/css" href="/static/css/style.css">
         <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.13/css/select2.min.css" rel="stylesheet" />
+        <link href="https://unpkg.com/tabulator-tables@6.2.5/dist/css/tabulator.min.css" rel="stylesheet">
```

Comment:
Looks like Tabulator also has a [bootstrap 3 theme](https://tabulator.info/docs/6.2/theme#framework-boot3); that will probably look better in the interface.

Reply:
Cool, it looks much better with it indeed!